<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>677</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>7</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>SDSi Test</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff/>
   </iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>48</width>
    <height>48</height>
   </size>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="margin">
       <number>5</number>
      </property>
      <item>
       <widget class="OrganismWidget" name="organismWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout">
        <property name="spacing">
         <number>3</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="pushButtonRun">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../res.qrc">
                <normaloff>:/icons/icons/control_play.png</normaloff>:/icons/icons/control_play.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../res.qrc">
                <normaloff>:/icons/icons/control_record.png</normaloff>:/icons/icons/control_record.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonStop">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../res.qrc">
                <normaloff>:/icons/icons/control_stop.png</normaloff>:/icons/icons/control_stop.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonReset">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../res.qrc">
                <normaloff>:/icons/icons/control_start.png</normaloff>:/icons/icons/control_start.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Step:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonStep1">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <pointsize>6</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>1</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonStep10">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <pointsize>6</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>10</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonStep50">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <pointsize>6</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>50</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBoxStopAtEvent">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Stop At Event?</string>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QSlider" name="frameSlider">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximum">
             <number>1</number>
            </property>
            <property name="tracking">
             <bool>true</bool>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="invertedAppearance">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLineEdit" name="lineEditStatus">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <family>Calibri</family>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>ForbiddenCursor</cursorShape>
              </property>
              <property name="text">
               <string>stable</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditTime">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditMsSim">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>sim: 0</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonInfo">
              <property name="text">
               <string>Info</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_2">
              <property name="text">
               <string>inspector</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_3">
              <property name="text">
               <string>comments</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_4">
              <property name="text">
               <string>params</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>701</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
   </widget>
   <widget class="QMenu" name="menuMisc">
    <property name="title">
     <string>Divide</string>
    </property>
    <addaction name="actionDivide_Highest_Along"/>
    <addaction name="actionDivide_Surface_Along"/>
    <addaction name="actionDivide_Highest_Away"/>
    <addaction name="actionDivide_Random"/>
    <addaction name="actionDivide_All"/>
   </widget>
   <widget class="QMenu" name="menuComplexify">
    <property name="title">
     <string>Complexify</string>
    </property>
    <addaction name="actionComplexify_All"/>
   </widget>
   <widget class="QMenu" name="menuPresets">
    <property name="title">
     <string>Presets</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuMisc"/>
   <addaction name="menuComplexify"/>
   <addaction name="menuPresets"/>
  </widget>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionDivide_Highest_Along">
   <property name="text">
    <string>Divide Highest Along</string>
   </property>
  </action>
  <action name="actionDivide_Surface_Along">
   <property name="text">
    <string>Divide Surface Along</string>
   </property>
  </action>
  <action name="actionDivide_Highest_Away">
   <property name="text">
    <string>Divide Highest Away</string>
   </property>
  </action>
  <action name="actionDivide_Random">
   <property name="text">
    <string>Divide Random</string>
   </property>
  </action>
  <action name="actionDivide_All">
   <property name="text">
    <string>Divide All</string>
   </property>
  </action>
  <action name="actionComplexify_All">
   <property name="text">
    <string>ComplexifyAll</string>
   </property>
  </action>
  <action name="actionPresets">
   <property name="text">
    <string>Presets</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OrganismWidget</class>
   <extends>QWidget</extends>
   <header>organismwidget.h</header>
   <container>1</container>
   <slots>
    <signal>cellSelected(Cell*)</signal>
    <signal>faceSelected(Face*)</signal>
    <signal>vertexSelected(Vertex*)</signal>
    <signal>edgeSelected(Edge*)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>organismWidget</sender>
   <signal>cellSelected(Cell*)</signal>
   <receiver>MainWindow</receiver>
   <slot>selectCell(Cell*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>organismWidget</sender>
   <signal>edgeSelected(Edge*)</signal>
   <receiver>MainWindow</receiver>
   <slot>selectEdge(Edge*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>393</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>392</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>organismWidget</sender>
   <signal>faceSelected(Face*)</signal>
   <receiver>MainWindow</receiver>
   <slot>selectFace(Face*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>2</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>organismWidget</sender>
   <signal>vertexSelected(Vertex*)</signal>
   <receiver>MainWindow</receiver>
   <slot>selectVertex(Vertex*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>80</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonRun</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>start()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>602</y>
    </hint>
    <hint type="destinationlabel">
     <x>614</x>
     <y>626</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonStep1</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>step1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>602</y>
    </hint>
    <hint type="destinationlabel">
     <x>614</x>
     <y>625</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonStep10</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>step10()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>602</y>
    </hint>
    <hint type="destinationlabel">
     <x>614</x>
     <y>622</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonStep50</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>step50()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>418</x>
     <y>602</y>
    </hint>
    <hint type="destinationlabel">
     <x>614</x>
     <y>624</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonStop</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>stop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>602</y>
    </hint>
    <hint type="destinationlabel">
     <x>614</x>
     <y>625</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonReset</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>602</y>
    </hint>
    <hint type="destinationlabel">
     <x>614</x>
     <y>626</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>record()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>602</y>
    </hint>
    <hint type="destinationlabel">
     <x>541</x>
     <y>503</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Open</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>load()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDivide_All</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>divideAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDivide_Highest_Along</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>divideHighest()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDivide_Highest_Away</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>divideHighestAway()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDivide_Random</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>divideRandomCell()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDivide_Surface_Along</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>divideAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showInspector()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>521</x>
     <y>659</y>
    </hint>
    <hint type="destinationlabel">
     <x>569</x>
     <y>674</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showComments()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>602</x>
     <y>659</y>
    </hint>
    <hint type="destinationlabel">
     <x>640</x>
     <y>651</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxStopAtEvent</sender>
   <signal>stateChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>stopAtEventChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>550</x>
     <y>598</y>
    </hint>
    <hint type="destinationlabel">
     <x>406</x>
     <y>670</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showSimulationParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>683</x>
     <y>659</y>
    </hint>
    <hint type="destinationlabel">
     <x>638</x>
     <y>584</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>frameSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>chooseFrame(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>24</x>
     <y>621</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>620</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionComplexify_All</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>complexifyAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectCell(Cell*)</slot>
  <slot>selectEdge(Edge*)</slot>
  <slot>selectFace(Face*)</slot>
  <slot>selectVertex(Vertex*)</slot>
  <slot>start()</slot>
  <slot>step1()</slot>
  <slot>step10()</slot>
  <slot>step50()</slot>
  <slot>stop()</slot>
  <slot>reset()</slot>
  <slot>load()</slot>
  <slot>propertiesSelected()</slot>
  <slot>divideHighestAway()</slot>
  <slot>divideAll()</slot>
  <slot>divideHighest()</slot>
  <slot>divideAlongBoundary()</slot>
  <slot>divideRandomCell()</slot>
  <slot>infoListVertices()</slot>
  <slot>infoListEdges()</slot>
  <slot>infoListFaces()</slot>
  <slot>infoListTetras()</slot>
  <slot>stopAtEventChanged(int)</slot>
  <slot>info()</slot>
  <slot>record()</slot>
  <slot>showInspector()</slot>
  <slot>showComments()</slot>
  <slot>setStopAtEvent(bool)</slot>
  <slot>showSimulationParameters()</slot>
  <slot>chooseFrame(int)</slot>
  <slot>complexifyAll()</slot>
 </slots>
</ui>
